### PR TITLE
Deployment response change for referenceDesignator.

### PR DESCRIPTION
**Pull request by ednad**

### CODE REVIEW

**Comment by oceanzus** [comment]
@DanielJMaher this is tested and ready for merge.  Critical fix to the deployments page.

File: ooiservices/app/uframe/asset_tools.py [hunk 1]
```diff
@@ -109,11 +98,12 @@ def get_assets_payload():
         # Get asset_list
         data, _ = new_compile_assets(result, compile_all=True)
         print '\n-- Completed loading assets....'
-        if not data:
+        if not data or data is None:
             message = 'Unable to process uframe assets; error creating assets_list.'
             raise Exception(message)
 
         # Cache 'asset_list'.
+        cache.delete('asset_list')
```
Comment (DanielJMaher) [inline ooiservices/app/uframe/asset_tools.py]: Moving the delete down here from lines 96/98 and then setting the cache on the next line is the "Hot Swap" methodology we had discussed.  right @ednad ?

**Comment by DanielJMaher** [comment]
@oceanzus I see several times where we are removing the array, node, instrument breakdown of ref_des.  Just to make sure, that isn't changing any functionality currently in the system.  right?

**Comment by ednad** [comment]
yes, that is the 'hot swap' we discussed.

**Pull request merged**